src/components/dialogs/network-modifications/line-split-with-voltage-level/line-split-with-voltage-level-illustration.tsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useIntl } from 'react-intl';
+import { AccordionIllustration } from '../widget/AccordionIllustration';
+import React, { useEffect, useState } from 'react';
+import { ReactComponent as LineSplitDiagram } from '../../../../images/network-modifications/illustrations/line-split-with-voltage-level.svg';
+import { useSelector } from 'react-redux';
+import { ReduxState } from '../../../../redux/reducer.type';
+import { PARAM_THEME } from '../../../../utils/config-params';
+import Paper from '@mui/material/Paper';
+
+export const styles = {
+    illustration: {
+        backgroundColor: 'transparent',
+        backgroundImage: 'none',
+        boxShadow: '0',
+        padding: '0px 0px',
+    },
+};
+
+/**
+ * This component displays a hide able explanation illustration
+ */
+const LineSplitWithVoltageLevelIllustration = () => {
+    const [showDiagram, setShowDiagram] = useState(true);
+    const intl = useIntl();
+    const theme = useSelector((state: ReduxState) => state[PARAM_THEME]);
+
+    useEffect(() => {
```

Comment:
maybe it is better to use memorization , because accessing to DOM is very heavy in javascript and using useEffect will access the DOM everytime clicking on "Information" Accordion.

Reply:
I added intl to the dependencies as you advised me.

Reply:
you forgot to add  replacedTexts

Reply:
Actually I did add it but then I add the following eslint warning message : 

'React Hook useEffect has an unnecessary dependency: 'replacedTexts'. Either exclude it or remove the dependency array. Outer scope values like 'replacedTexts' aren't valid dependencies because mutating them doesn't re-render the component'

Given that replacedTexts is a const oustide the component it kind of make sense but I am going to read the doc more closely.